src/constants/translations/en/student-home-page.json

```diff
@@ -1,8 +1,10 @@
 {
   "findTutorBlock": {
-    "title": "A Global Network of Tutors",
+    "title": {
+      "student": "A Global Network of Tutors"
+    },
```

Comment:
If you change it here you need to change the same in Ukrainian language.
Also when you have a tutor role, you will see the it is broken. Add text for the tutor role.

Reply:
fixed